tests/unit/test_handlers.py

```diff
@@ -1175,49 +1148,6 @@ def test_non_ascii_characters(self):
         )
 
 
-class TestRetryHandlerOrder(BaseSessionTest):
```

Comment:
Just to make sure we're on the same page: Was this test removed because we now register the handler to the `before-parse` event and guarantees it gets called before the retry logic?

Reply:
Yes